org.eclipse.jdt.core/codeassist/org/eclipse/jdt/internal/codeassist/DOMCompletionUtil.java

```diff
@@ -45,4 +46,31 @@ public static AbstractTypeDeclaration findParentTypeDeclaration(ASTNode nodeToSe
 		return (AbstractTypeDeclaration) DOMCompletionUtil.findParent(nodeToSearch, new int[] { ASTNode.TYPE_DECLARATION, ASTNode.ENUM_DECLARATION, ASTNode.RECORD_DECLARATION, ASTNode.ANNOTATION_TYPE_DECLARATION });
 	}
 
+	private static final List<String> JAVA_MODIFIERS = List.of(
+				"public", //$NON-NLS-1$
+				"private", //$NON-NLS-1$
+				"static", //$NON-NLS-1$
+				"protected", //$NON-NLS-1$
+				"sychronized", //$NON-NLS-1$
+				"abstract", //$NON-NLS-1$
+				"final", //$NON-NLS-1$
+				"default", //$NON-NLS-1$
+				"native", //$NON-NLS-1$
+				"strictfp", //$NON-NLS-1$
+				"transient", //$NON-NLS-1$
+				"volatile" //$NON-NLS-1$
```

Comment:
Those should reference `ModifierKeyword.KEYWORD_xxx.keyword` instead of duplicating the value.